Composer/packages/lib/code-editor/src/lg/modalityEditors/StringArrayEditor.tsx

```diff
@@ -118,7 +118,11 @@ export const StringArrayEditor = React.memo(
 
     useEffect(() => {
       const keydownHandler = (e: KeyboardEvent) => {
-        if (submitKeys.includes(e.key)) {
+        if (
+          e.key === 'Enter' ||
```

Comment:
Are there no defined constants for keys to compare with?  (C# has KeyCodes, but I'm not sure about React)

Reply:
key"codes" are deprecated
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode